src/Calendar.tsx

```diff
@@ -0,0 +1,13 @@
+import './Calendar.css';
+
+const Calendar = (props: any) => {
```

Comment:
don't use 'any' type if you can. Here you can describe type, as 'classes' object is a part of Material UI module it probably has defined it's types. If types of imported module are not available after install (and often they are not) try install types with 'npm install @types/package-name'.

Reply:
![obraz](https://user-images.githubusercontent.com/7761841/106622983-70eb1b80-6574-11eb-9e00-e92ed64b1dc5.png)